studio-docs/source/data-privacy.md

```diff
@@ -8,6 +8,31 @@ This article describes which data is and is _not_ sent to Apollo as part of its
 
 > Most importantly, result data from GraphQL operations that your server executes is **never** sent to Apollo.
 
+## What technology sends data to Apollo Studio
```

Comment:
Maybe this is more like "How Apollo's open source tools connect to Apollo Studio"